CMakeLists.txt

```diff
@@ -36,9 +36,26 @@ set(NULL_ENTROPY_WARNING "${WARNING_BORDER}"
                          "${NULL_ENTROPY_WARN_L3}"
                          "${WARNING_BORDER}")
 
+set(CTR_DRBG_128_BIT_KEY_WARN_L1 "****  WARNING!  MBEDTLS_CTR_DRBG_USE_128_BIT_KEY defined!\n")
+set(CTR_DRBG_128_BIT_KEY_WARN_L2 "****  Using 128-bit keys for CTR DRBG limits the security of all generated keys\n")
+set(CTR_DRBG_128_BIT_KEY_WARN_L3 "****  and all operations that use random values generated to 128-bit security\n")
+
+set(CTR_DRBG_128_BIT_KEY_WARNING "${WARNING_BORDER}"
+                         "${CTR_DRBG_128_BIT_KEY_WARN_L1}"
+                         "${CTR_DRBG_128_BIT_KEY_WARN_L2}"
+                         "${CTR_DRBG_128_BIT_KEY_WARN_L3}"
+                         "${WARNING_BORDER}")
+
 find_package(Perl)
 if(PERL_FOUND)
 
+    # If NULL Entropy is configured, display an appropriate warning
```

Comment:
Copy-paste error: this comment should say "If CTR_DRBG is configured to use 128-bit keys" or something like that.